listening.js

```diff
@@ -0,0 +1,6 @@
+const app = require("app");
+
+app.listen(9090, (err) => {
+  if (err) console.log(err);
+  console.log("9090 is listening...");
```

Comment:
For the time being this isn't strictly necessary as it isn't used in testing, but it's okay to include just in case you wanted to run insomnia to probe the endpoints